policyengine_us/parameters/gov/states/ne/tax/income/rates/head_of_household.yaml

```
@@ -1,37 +1,61 @@
-description: NE income tax rates for head-of-household filing units
+description: Nebraska income tax rates for head-of-household filing units
```

and for the other filing statuses
```suggestion
description: Nebraska taxes head of household filers at these rates.
```